nuxeo-drive-core/src/main/resources/OSGI-INF/nuxeodrive-listeners.xml

```diff
@@ -48,6 +49,14 @@
       class="org.nuxeo.drive.listener.NuxeoDriveVirtualEventLogger" />
 
     <documentation>
+      Handle group change events fired by the UserManager.
+
+      @author Antoine Taillefer
+    </documentation>
+    <listener name="nuxeoDriveGroupUpdateListener" async="true" postCommit="true"
+      class="org.nuxeo.drive.listener.NuxeoDriveGroupUpdateListener" />
```

Comment:
The listener should contain the list of `<event>` it's interested in. In which case there's no need for the `acceptedEvents.contains(event.getName())` in the listener code, and things are much faster anyway because the listener doesn't need to receive all events and then filter by hand.